### ISSUE TITLE
[v3] Features that are not yet / will be supported?

### ISSUE THREAD

**Issue by IljaDaderko** [fullstackreact/react-native-firestack#198]
By the quick glance of README file I'm not entirely sure if features like Crash reporting / notifications etc.. are supported by firestack, this led to a question:

Is there a list somewhere that shows current firebase features put against current firestack features and some sort of indication if missing fireabse features will be supported at certain point?

If there is no such list would you consider a pr from me for this? Could create a table given I have list of current + planned firestack features :)

**Comment by auser** [comment]
Yes, absolutely. Send a PR and I'll fill it out!

:)

**Comment by Salakar** [comment]
@IljaDaderko before you jump ahead and do that, please check out the v3 branch - that's the cutting edge release ;p lots of new things implemented there such as analytics, mirroring fb web sdk and so on. 

If you'd like to PR for this then please send the PR to my fork, it'll get merged in then synced up to v3 branch.

**Comment by IljaDaderko** [comment]
@Salakar @auser sorry for late response guys, will gladly jump on this 👍 I think I'll have PR ready next weekend, due to new year celebrations. @Salakar I assumed v3 was merged into master already (As both documentations have exactly same installation guides for npm), what are the differences between these at the moment?

**Comment by Salakar** [comment]
@IljaDaderko so many changes to go through all of them, I think the only things left now that are missing from firestack are:

- Database transactions - don't think this is doable over RNBridge.
- Complete FCM implementation - just the basics in, onNotif, sendToTopic etc.
- Crash reporting - planned.
- Ads - this is a big one though - bigger than all of them combined really - so going to be doing a separate repo for it
- Remote config - we have an implementation but it needs polishing so not pushing it yet.
- App Indexing - another one that I don't think is doable over RNBridge.

Eventually it'll just be those two features that are not doable over the react native bridge.

Master and v3 have not been merged.

**Comment by Salakar** [comment]
@IljaDaderko its very usable, we use it in our prod app, its stable atm, my fork is where the madness is at, then I push up to v3 branch at stable intervals.

Upgrade path is literally switch to the firebase web sdk syntax e.g. `.database` becomes `.database()` and lots of similar changes such as method name changes. Pretty much just follow web sdk docs.

**Comment by IljaDaderko** [comment]
@Salakar Awesome, I assume it is available via npm at `react-native-firestack@3.0` or something like this?

**Comment by Salakar** [comment]
@IljaDaderko not yet, you can npm install it with the git name / url

e.g. 
```bash
npm i fullstackreact/react-native-firestack#v3
```

@auser said he'll release a v3 alpha version to npm sometime tomorrow.

**Comment by Salakar** [comment]
@IljaDaderko still waiting on @auser sorry.

**Comment by auser** [comment]
I haven't released it yet. I'm converting the example app (FirestackApp) to use v3 (in the v3 branch there). Once I've completed the transition, I'll release an alpha version on npm. 

https://github.com/fullstackreact/FirestackApp/tree/feature/v3

Pos are gladly accepted!

**Comment by IljaDaderko** [comment]
https://github.com/fullstackreact/react-native-firestack/pull/223 needs to be filled out

**Comment by IljaDaderko** [comment]
still needs filling out

**Comment by nkashyap** [comment]
Hello guys, if you push `Remote config` changes in the branch I can pick it up. Thanks
